web/javascript/inspector.js

```diff
@@ -301,24 +301,34 @@ function Inspector(controller) {
             // last activity
             //
 
+            str = '';
             latest = -1;
             if (torrents.length < 1) {
                 str = none;
             } else {
                 baseline = torrents[0].getLastActivity();
                 for (i = 0; t = torrents[i]; ++i) {
                     d = t.getLastActivity();
-                    if (latest < d) {
+                    if (baseline != d) {
+                        str = mixed;
```

Comment:
Not sure "mixed" should implemented here. If multiple torrents are selected displaying the latest activity looks reasonable. It provide some useful and meaningful information anyway.

Reply:
I wanted to make it more consistent with other fields that way. For example when you select multiple torrents the "Running Time" you get is a "mixed" instead of the one than run the longest, so I decided to do the same for "Last Activity". Wouldn't selecting a "never" active torrent with another one that was active some time ago and getting reported activity time be more misleading?

Reply:
I was thinking about next scenarios. Suppose I have a few douzens of stalled torrents. I can simply select all of them and see what is the latest time they were active. 

Ok. It is just a two ways to look on the feature. 
Just commented. Descision is yours. :)